autoload/fruzzy/ctrlp.vim

```diff
@@ -0,0 +1,52 @@
+"
+" Sources:
+" - https://github.com/nixprime/cpsm
+" - https://sourcegraph.com/github.com/nixprime/cpsm@master/-/blob/autoload/cpsm.py#L27
+" - https://github.com/raghur/fruzzy/issues/1
+"
+if has('python3')
+  let pyfile = 'py3file '
+  let py = 'py3eval'
+elseif has('python')
```

Comment:
I don't  intend to support py2 - I definitely haven't tested fruzzy or the native extn against it so you can just go with py3

Reply:
Great, that makes everything simpler. :)